src/pages/HomeView.jsx

```diff
@@ -1,5 +1,19 @@
+import styled from 'styled-components';
+import VideosList from '../components/videos/VideosList';
+import mockData from '../helper/mock-data.json';
+
+const Container = styled.div`
+  padding: 15px 10px;
+`;
+
 const HomeView = () => {
-  return <main role="main">Home View</main>;
+  const videos = mockData.items;
+  videos.shift();
```

Comment:
**Question**: 

What is the idea behind this `.shift()`?  I think this could be removed

Reply:
I used it to discard the first element of the mock-data items, so that it would render the same data as the demo application. But yes, it is obviously unnecessary to do so. Thank you.